spec/core/ics-002-client-semantics/README.md

```diff
@@ -536,12 +583,13 @@ Calling `createClient` with the client type and initial consensus state creates
 ```typescript
 function createClient(
```

Comment:
This function in general is out of date. Since we take in a ClientState now.

I think if we refactored this to follow what ibc-go is doing this can be achieved.

Since the clientDependencies will be fields on the ClientState